build.sc

```diff
@@ -172,12 +176,14 @@ object openapi extends BaseScalaModule {
 
   object tests extends this.ScalaTests with BaseMunitTests {
     def ivyDeps = super.ivyDeps() ++ Agg(
-      Deps.smithy.build
+      Deps.smithy.build,
+      Deps.scalaJavaCompat
     )
   }
 }
 
-object cli extends BaseScalaModule with buildinfo.BuildInfo {
+object cli extends Cross[CliModule](scalaVersions)
```

Comment:
arguably, we don't really need to cross-build this one as CLI usage, as CLIs shouldn't be consumed as libraries.

Reply:
smithy-lang/smithy enters the room lol

Reply:
loll, sure but it's a self contained fat jar
so cross built or not it would not matter

but I was thinking they would have dropped it by now